src/audio/pulseaudio.rs

```diff
@@ -15,7 +14,9 @@ use libpulse_binding::context::{
 use libpulse_binding::mainloop::standard::IterateResult;
 use libpulse_binding::mainloop::standard::Mainloop;
 use libpulse_binding::proplist::{properties, Proplist};
-use libpulse_binding::volume::{ChannelVolumes, VOLUME_MAX, VOLUME_NORM};
+use libpulse_binding::volume::{ChannelVolumes, Volume, VOLUME_NORM};
+
+static MAX_VOLUME: Volume = Volume(131072); // 2^16 * 2 = 200%
```

Comment:
Any idea if there's a more adequate value? I was surprised to see pulse audio allow the volume to go up to 2000% and more, but now that I check pacmd doesn't have a limit either.

Reply:
Honestly, I think any limit we'd be setting is arbitrary, so I'm tempted to say that there shouldn't be a limit.

Reply:
Then maybe there should be an optional argument to add to set the cap. Something like:

```
platformctl audio -- +0.10 1.5
```

To increase volume by 10% up to 150% max.

Reply:
That sounds reasonable.